iam.tf

```diff
@@ -92,7 +92,7 @@ resource "aws_iam_policy" "additional_json" {
 }
 
 resource "aws_iam_role_policy_attachment" "additional_json" {
-  for_each = var.lambda_policy_json != null ? local.lambdas : {}
+  for_each = var.create_lambda_policy != null ? local.lambdas : {}
```

Comment:
Checking for `!= null` here is probably wrong, since the variable type is `bool`:
```
true  != null  => true
false != null  => true
```

Reply:
You are 100% right on that - I've fixed it now :)